test/apiv2/20-containers.at

```diff
@@ -113,18 +113,18 @@ t GET libpod/containers/json?last=1 200 \
 cid=$(jq -r '.[0].Id' <<<"$output")
 
 if root; then
-    t GET libpod/containers/stats?containers='[$cid]' 200
+    t GET libpod/containers/$cid/stats?stream=false 200
 else
     if have_cgroupsv2; then
-        t GET libpod/containers/stats?containers='[$cid]' 200
+	t GET libpod/containers/$cid/stats?stream=false 200
     else
-        t GET libpod/containers/stats?containers='[$cid]' 409
+	t GET libpod/containers/$cid/stats?stream=false 409
```

Comment:
These are two different endpoints, I would suggest you keep testing the endpoint you actually fixed and juts fix the tests instead.
The correct URl param would be `?containers=$cid`
And then add some basic output checks as well.